docs/declarative-customization/list-form-conditional-show-hide.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Show or hide columns in a list form
+description: Customize which columns to show or hide using a conditional formula in the list form by constructing a simple formula that are equations that perform conditional checks on values in a SharePoint list or library. 
```

Comment:
Customize which columns to show or hide using a conditional formula in the list form by constructing a simple formula that are equations performing conditional checks on values in a SharePoint list or library.